talekclient panic due to uninitialized Handle.hasher

```
27fb29696b31:/build/talek/cli/talekclient#  ./talekclient --verbose --config /path/to/config.json --topic talek.foo --write "hellooo"
Connection to RPC established.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x6917f4]

goroutine 1 [running]:
github.com/privacylab/talek/libtalek.(*Handle).nextInterestVector(0xc000016290)
	/build/talek/libtalek/handle.go:92 +0xb4
github.com/privacylab/talek/libtalek.(*Topic).GeneratePublish(0xc000016280, 0x3b0?, {0xc000146400, 0x3b0, 0x3b0})
	/build/talek/libtalek/topic.go:87 +0xcc
github.com/privacylab/talek/libtalek.(*Client).Publish(0xc000000180, 0x7fff4b9e6e66?, {0xc00001ce10?, 0x1?, 0x1?})
	/build/talek/libtalek/client.go:126 +0x230
main.main()
	/build/talek/cli/talekclient/main.go:92 +0xca5
```

--- a/libtalek/handle.go
+++ b/libtalek/handle.go
@@ -53,7 +53,6 @@ type Handle struct {
 //NewHandle creates a new topic handle, without attachment to a specific topic.
 func NewHandle() (h *Handle, err error) {
 	h = &Handle{}
-	h.hasher = sha256.New()
 	err = initHandle(h)
 	return
 }
@@ -244,6 +243,9 @@ func (h *Handle) UnmarshalText(text []byte) error {
 		return err
 	}
 	if err := h.Seed2.UnmarshalBinary(s2); err != nil {
+		return err
+	}
+	if err := initHandle(h); err != nil {
 		return err
 	}
 	return nil

